### ISSUE TITLE
508: skip navigation

### ISSUE THREAD

**Issue by leahbannon** [18F/fbopen#102]
13A-D- If there is any content that is repeated on different pages of a site, a skip navigation link should be provided to jump users past it. This link can be hidden as long as its revealed when a keyboard user first hits tab.
